widgets/features/styles/default.less

```diff
@@ -134,6 +166,18 @@
 			}
 		}
 
+		.sow-features-feature-right-left-container {
+			display: flex;
+			flex-direction: inherit;
+			gap: 12px;
```

Comment:
Is this gap value intentional? This will result in extra spacing. Compare "Top / bottom-align off / more yes" before and after.